skopt/gbt.py

```diff
@@ -50,4 +50,4 @@ def fit(self, X, y):
 
     def predict(self, X):
         """Predictions for each quantile."""
-        return np.vstack([rgr.predict(X) for rgr in self.regressors_])
+        return np.asarray([rgr.predict(X) for rgr in self.regressors_])
```

Comment:
This is not consistent with the scikit-learn API. In my opinion, we should see that as a multi-output estimator, hence the output shape should be `len(X), n_outputs`.